### PR TITLE
[SOFT-459] add dbc releaser action

### PR DESCRIPTION
**Pull request by benlin1221**
Creates a release containing the system_can.dbc file under tag [action run #]-[commit id], for every push to master

### CODE REVIEW

File: .github/workflows/make_dbc_release.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+      - name: Build codegen & make release
+        run: |
+          make codegen
+          make codegen_dbc
+          echo ${{ secrets.GITHUB_TOKEN }} > github_token.txt
+          gh auth login --with-token < github_token.txt
+          CUR_COMMIT=$(git rev-parse master)
+          echo "Creating release under tag ${GITHUB_RUN_NUMBER}-${CUR_COMMIT}"
+          gh release create "${GITHUB_RUN_NUMBER}-${CUR_COMMIT}" codegen/system_can.dbc
```
Comment (ryandancy) [inline .github/workflows/make_dbc_release.yml:74]: One request, could you prefix the release name with `DBC-` so it's more obvious what the release is?

File: .github/workflows/make_dbc_release.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
+
+on:
+  #run on pushes to master
+  push:
+    branches: [ master ]
+
+jobs:
+  make_dbc:
+    #not sure what to put here, referenced main.yml
```
Comment (ryandancy) [inline .github/workflows/make_dbc_release.yml:9]: Yep this looks good! @abdullahs26 when you get main.yml updated to a higher ubuntu and it's working, could you then upgrade this script?

**Review by ryandancy — CHANGES_REQUESTED** [review]
Looks great, just one request :)

File: .github/workflows/make_dbc_release.yml (hunk 2 above)
Comment (abdullahs26) [reply .github/workflows/make_dbc_release.yml:9]: Yeah sure, sounds good.

**Review by ryandancy — APPROVED** [review]

**Pull request merged**